lib/workers/repository/index.ts

```diff
@@ -53,7 +59,9 @@ export async function renovateRepository(
       const res = await updateRepo(config, branches);
       setMeta({ repository: config.repository });
       addSplit('update');
-      await setBranchCache(branches);
+      if (performExtract) {
+        await setBranchCache(branches); // update branch cache if performed extraction
```

Comment:
It should. This part of code isn't related to the branch cache changes. 
@Gabriel-Ladzaretti My understanding is that you don't want to call `setBranchCache` when onboarding to prevent the branch info from being stored right?

Reply:
Needs confirmation